.travis.yml

```diff
@@ -13,12 +13,15 @@ env:
   - TEST_DIR=joda-time-joda-time
   - TEST_DIR=org.joda-joda-primitives
   - TEST_DIR=org.apache.commons-commons-lang3
+  - TEST_DIR=com.fasterxml.jackson.core
 
 # Uncomment to add a library that should fail to build, along with documentation on why.
-# matrix:
-#   allow_failures:
-#     # Blocked on (github issue url)
-#     - env: TEST_DIR=libraryBuilds/com.example-library
+matrix:
+  allow_failures:
+      # Blocked on (github issue url)
+#     - env: TEST_DIR=com.example-library
+      # Blocked on: https://github.com/j2objc-contrib/j2objc-common-libs-e2e-test/issues/48
```

Comment:
Actually once this is submitted. Issue 48 should be closed as it now has a continuous build. You should have a separate bug explaining the build failure and then reference that on line 23 instead.